.circleci/config.yml

```diff
@@ -94,6 +94,13 @@ jobs:
           target: test-sim-multi-seed-long
           description: "Test multi-seed simulation (long)"
 
+  test-cli:
```

Comment:
could we make this a  github action. 

we are trying to migrate away from circle